tasks/base_tasks.py

```diff
@@ -1735,3 +1735,136 @@ def complete(self):
 
     def output(self):
         return LocalTarget(self._get_filepath())
+
+
+class BaseInterpolationTask(TableTask):
+    def requires(self):
+        '''
+        This method MUST be overriden in subclasses.
+        Subclasses MUST define the following requirements:
+
+        'source_geom_columns'   a task with the source geometry columns
+        'source_geom'           a task with the source geometries (interpolated from)
+        'source_data_columns'   a task with the source data columns
+        'source_data'           a task with the source data (interpolated from)
+        'target_geom_columns'   a task with the target geometry columns
+        'target_geom'           a task with the source geometries (interpolated to)
+        'target_data_columns'   a task with the target data columns
+        '''
+        raise NotImplementedError('The requires method must be overriden in subclasses')
+
+    def get_interpolation_parameters(self):
```

Comment:
Maybe documenting that it should return a dictionary with those keys would be interesting.